.circleci/config.yml

```diff
@@ -42,16 +42,16 @@ jobs:
           fi
 
 
-#  e2eTestCurMasterClusterState:
-#    environment:
-#      TESTED_VERSION: "current"
-#      TEST_DIR: "integration/test/clusterstate"
-#    <<: *e2eTest
-#  e2eTestCurPRClusterState:
-#    environment:
-#      TESTED_VERSION: "current"
-#      TEST_DIR: "integration/test/clusterstate"
-#    <<: *e2eTest
+  e2eTestCurMasterClusterState:
+    environment:
+      TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/clusterstate"
+    <<: *e2eTest
+  e2eTestCurPRClusterState:
+    environment:
+      TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/clusterstate"
+    <<: *e2eTest
```

Comment:
In the beginning we disabled these because there were versions which were not able to comply with the tests. We moved on and now we can actually use them.